lib/web_ui/lib/src/engine/clipboard.dart

```diff
@@ -15,8 +15,12 @@ class ClipboardMessageHandler {
       PasteFromClipboardStrategy();
 
   /// Handles the platform message which stores the given text to the clipboard.
-  void setDataMethodCall(MethodCall methodCall) {
-    _copyToClipboardStrategy.setData(methodCall.arguments['text']);
+  void setDataMethodCall(
+      MethodCall methodCall, ui.PlatformMessageResponseCallback callback) {
+    _copyToClipboardStrategy.setData(methodCall.arguments['text']).then((_) {
+      const MethodCodec codec = JSONMethodCodec();
+      callback(codec.encodeSuccessEnvelope(true));
+    }).catchError((error) => print('Could not copy text: $error'));
```

Comment:
There are two concerns I have here:
1. We are printing the errors in multiple places.
2. We are not responding to the platform message when there's an error.

Reply:
Thanks, I removed the extra message. I am also calling `encodeErrorEnvelope` now for the error case.
I also added the error case to the paste.